editor/components/page-attributes/parent.js

```diff
@@ -43,31 +37,30 @@ export function PageAttributesParent( { parent, postType, items, onUpdateParent
 	);
 }
 
-const applyConnect = connect(
-	( state ) => {
-		return {
-			postId: getCurrentPostId( state ),
-			parent: getEditedPostAttribute( state, 'parent' ),
-			postTypeSlug: getCurrentPostType( state ),
-		};
-	},
-	{
-		onUpdateParent( parent ) {
-			return editPost( { parent: parent || 0 } );
-		},
-	}
-);
+const applyWithSelect = withSelect( ( select ) => {
+	const { getPostType } = select( 'core' );
+	const { getCurrentPostId, getEditedPostAttribute } = select( 'core/editor' );
+	const postTypeSlug = getEditedPostAttribute( 'type' );
+	return {
+		postId: getCurrentPostId(),
+		parent: getEditedPostAttribute( 'parent' ),
+		postType: getPostType( postTypeSlug ),
```

Comment:
`postType` contains `slug`, so there is no need to pass it as a separate prop.

Reply:
But postType is not necessarily loaded, that was my thinking.

Reply:
`postTypeSlug` - the same applies. We can leave it as it is :)